client/quantum_serverless/core/files.py

```diff
@@ -90,3 +90,17 @@ def list(self) -> List[str]:
                 )
             )
         return response_data.get("results", [])
+
+    def delete(self, file: str) -> Optional[str]:
+        """Deletes file uploaded or produced by the programs,"""
+        tracer = trace.get_tracer("client.tracer")
+        with tracer.start_as_current_span("files.delete"):
+            response_data = safe_json_request(
+                request=lambda: requests.get(
+                    f"{self.host}/api/{self.version}/files/delete/",
```

Comment:
I would like to propose here the next URI:

```
f"{self.host}/api/{self.version}/files/",
```

Similar to what we are doing with the `list` method. The difference would be the HTTP method and we would be doing an approximation to the API Rest URI nomenclature standard. We are not following it strictly because our API doesn't adjust to it but it's a good practice in general I would say.

Reply:
I tried it but it doesn't work because of the restriction of `viewsets.ModelViewSet` of `django`.

Reply:
Understood, no problem thanks @akihikokuroda 😄